src/views/ipv/page/update-details-failed.njk

```diff
@@ -11,20 +11,32 @@
 
 {% block content %}
   <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.updateDetailsFailed.header' | translate }}</h1>
-  <p class="govuk-body">{{ 'pages.updateDetailsFailed.content.paragraph1' | translate }}</p>
+  {% if context == "repeatFraudCheck" %}
+    <p class="govuk-body">{{ 'pages.updateDetailsFailed.content.paragraph1RepeatFraudCheck' | translate }}</p>
```

Comment:
you should be able to use `translateWithContext(context)` here and remove the if/else

Reply:
Done